app/views/admin/products_v3/_product_variant_row.html.haml

```diff
@@ -1,7 +1,8 @@
 = form.fields_for("products", product, index: product_index) do |product_form|
   %tbody.relaxed{ id: dom_id(product), data: { 'record-id': product_form.object.id,
                           controller: "nested-form product",
-                          action: 'rails-nested-form:add->bulk-form#registerElements rails-nested-form:remove->bulk-form#toggleFormChanged' } }
+                          action: 'rails-nested-form:add->bulk-form#registerElements rails-nested-form:remove->bulk-form#toggleFormChanged' },
+                  class: (defined?(should_slide_in) && should_slide_in) ? 'slide-in' : '' }
```

Comment:
If you rely on an optional variable then it's good to declare that at the top of the partial:
```haml
- should_slide_in ||= false
```

That makes the interface clearer and the code easier to read.

Reply:
That's nice. Sure, I'll incorporate it. Thanks 🙂